scripts/robo-collector-commit

```diff
@@ -82,9 +82,7 @@ info() {
 push_collector() {
     collector_repo_dir="$1"
 
-    # Redirect to devnull so this doesn't print the value of the token on
```

Comment:
To be clear - this redirection is no longer required on GHA because it will mask the secret value instead of printing it.